tests/cxx/namespace_use-d1.h

```diff
@@ -0,0 +1,12 @@
+//===--- namespace_use-d1.h - test input file for iwyu --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+namespace i1_ns {
+void action();
```

Comment:
Let's use Google style names for test symbols too. Functions are PascalCase, so this should be `Action`.